src/patch/parse.js

```diff
@@ -7,41 +7,38 @@ export function parsePatch(uniDiff, options = {}) {
     let index = {};
     list.push(index);
 
-    // Ignore any leading junk
-    while (i < diffstr.length) {
-      if (/^(Index:|diff -r|@@)/.test(diffstr[i])) {
+    // Parse diff metadata
+    for (; i < diffstr.length; i++) {
```

Comment:
Not going to accept purely stylistic changes for fear of the noise impact. Show me the substance not your own style. When working in the OSS world you have to remember that you aren't coding for yourself unless it's your own project.